pkg/settings/settings.go

```diff
@@ -33,7 +33,7 @@ var (
 	BackupTargetSet        = NewSetting(BackupTargetSettingName, InitBackupTargetToString())
 	RancherEnabled         = NewSetting("rancher-enabled", "false") // Specify whether the UI should display the Rancher UI navigation
 	UpgradableVersions     = NewSetting("upgradable-versions", "")
-	VersionMetadataURL     = NewSetting("version-metadata-url", "")
+	VersionMetadataURL     = NewSetting("version-metadata-url", "https://raw.githubusercontent.com/rancher/harvester/stable/version-metadata.json")
```

Comment:
I think we may not need this configuration on the `master` branch and it can be configured accordingly on the `v*` branches